src/doghouse/adapters/git/git_adapter.py

```
@@ -0,0 +1,40 @@
+import subprocess
+from typing import List
```

Ruff (0.15.7)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)